### ISSUE TITLE
MathType causes view-position-after-root erro being thrown when setting editor data with HTML+MathML

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#5765]
## 📝 Provide detailed reproduction steps (if any)

1. Create Classic CKEditor 5 with MathType plugin
2. In HTML page insert below code and run it in a browser:
```
ClassicEditor.create( document.querySelector( '#editor' ) )
	.then( editor => {
		window.editor = editor;
		window.editor.setData( 
			"<p style=\"margin-top:12pt; text-align:justify\"> <math xmlns=\"http://www.w3.org/1998/Math/MathML\">  <msub>   <mrow>    <mi mathcolor=\"000000\">     P    </mi>   </mrow>   <mrow>    <mi mathvariant=\"italic\" mathcolor=\"000000\">     fs    </mi>   </mrow>  </msub> </math></p> <p style=\"margin-top:12pt; text-align:justify\">  </p>"
		 );
	} ).catch( err => {
		console.error( err.stack );
	} );
```
**NOTE:** Please also read Other details section.

### ✔️ Expected result

Content should be set and no error should be thrown.

### ❌ Actual result

Content isn't set and there is error being thrown: `CKEditorError: view-position-after-root: You cannot make position after root`

![obraz](https://user-images.githubusercontent.com/5314687/69236245-691fa100-0b93-11ea-8db5-cac0b3bcab67.png)

## 📃 Other details

This error gets only thrown when MathType plugin is used. If you remove the MathType from your build, editor will process pasted content without any issues. 

* Browser: Any
* OS: Any
* CKEditor version: 15.0.0
* Installed CKEditor plugins: MathType

---
**NOTES:**

In the attached zip file there are files to create sample project (`package.json`, `ckeditor.js` and `index.html`) which helps to reproduce the problem - [files.zip](https://github.com/ckeditor/ckeditor5/files/3891678/files.zip).

The most important is `index.html` sample which tries to insert below code:
```
<p style=\"margin-top:12pt; text-align:justify\"> <math xmlns=\"http://www.w3.org/1998/Math/MathML\">  <msub>   <mrow>    <mi mathcolor=\"000000\">     P    </mi>   </mrow>   <mrow>    <mi mathvariant=\"italic\" mathcolor=\"000000\">     fs    </mi>   </mrow>  </msub> </math></p> <p style=\"margin-top:12pt; text-align:justify\">  </p>
```
What is weird is that error can be prevented by making different changes to above HTML.  If you remove last paragraph or spaces between `>     P    <` and `>     fs    <`, everything will work as expected.

Inside the sample `index.html` below the `setData` method you will also find original HTML (with spaces and `\n` characters) and slightly reduced one with removed spaces and `\n` characters. Both were divided into 4 lines (for better readability) where each is a paragraph. That HTML (whichever of two) when used in full also causes error however if you use a single paragraph, even the longest one with all the spaces and new line characters between tags, everything will work as expected.

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by FilipTokarski** [comment]
I checked it with the new version of MathType and this bug seems to be fixed.
When I open the editor with this data, I get a math formula rendered correctly and no errors.

**Comment by FilipTokarski** [comment]
This issue is fixed in the new MathType version 7.19.0
